setup.py

```diff
@@ -46,7 +50,7 @@
 
 setup(
     name='elixir-daisy',
-    version='1.4.0',
+    version='1.4.1',
```

Comment:
Probably the version could be already bumped

Reply:
version was bumped to 1.5.0